src/main/resources/scripts/helloWorldPerfTesting.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+##===========SETUP===============##
+sudo yum install -y java-17-amazon-corretto-devel && sudo yum -y update && sudo yum -y install docker && sudo yum -y install git && sudo yum remove -y python3.7 && sudo yum update -y && sudo yum groupinstall "Development Tools" -y && sudo yum erase openssl-devel -y && sudo yum install openssl11 openssl11-devel libffi-devel bzip2-devel wget -y
```

Comment:
helloWorldPerfTesting and anomalyDetectionTesting looks quite the same except few tweaks. Can we create a common file for all the common installations?